package.json

```diff
@@ -46,5 +46,8 @@
     "test:app": "jest --color --reporters=default --reporters=jest-junit --testPathPattern=/src/app",
     "test:web": "jest --color --reporters=default --reporters=jest-junit --testPathPattern=/src/web",
     "test:e2e": "jest --color --reporters=default --reporters=jest-junit --testPathPattern=/src/__tests__"
+  },
+  "dependencies": {
+    "node-forge": "^0.10.0"
```

Comment:
Added dependencies in the package file are not available in the deployed clients.
Use `CryptoJS.AES.decrypt` instead

Reply:
i had already changed to CryptoJS.AES.decrypt, but i can't get it to output anything useful